### ISSUE TITLE
JavaEESecCDIExtension.beforeBeanDiscovery uses deprecated calls

### ISSUE THREAD

**Issue by neilgsyoung** [OpenLiberty/open-liberty#1796]
While working on issue #1659 I noticed that when I bring up an OL server with appSecurity-3.0 and cdi-2.0 features I get the Weld messages,

```
W WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class com.ibm.ws.security.javaeesec.cdi.extensions.SecurityContextProducer is deprecated from CDI 1.1!
W WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class com.ibm.ws.security.javaeesec.cdi.extensions.AutoApplySessionInterceptor is deprecated from CDI 1.1!
W WELD-000146: BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) used for class com.ibm.ws.security.javaeesec.cdi.extensions.RememberMeInterceptor is deprecated from CDI 1.1!
```
As advised by @Emily-Jiang , the addAnnotatedType(AnnotatedType, String) call should be used instead.

**Comment by toshiyamamoto** [comment]
This will be resolved by PR #1906

**Comment by teddyjtorres** [comment]
Closing issue since PR #1906 was merged.
